src/containers/flows/MintDropNFT/index.tsx

```diff
@@ -360,11 +384,23 @@ const MintDropNFTFlowContainer = () => {
 
 	useAsyncEffect(async () => {
 		if (library) {
-			const price = await zSaleInstance.getSalePrice(library.getSigner());
+			const price = await zSaleInstance.getSalePrice();
 			setPricePerNFT(Number(price));
 		}
 	}, [zSaleInstance, library]);
 
+	useAsyncEffect(async () => {
+		const interval = setInterval(async () => {
+			if (Date.now() > PRIVATE_SALE_END_DATE) {
+				setCountdownDate(PUBLIC_SALE_START_TIME);
+				clearInterval(interval);
+			} else {
+				setCountdownDate(undefined);
+			}
+		}, 13000);
+		return () => clearInterval(interval);
+	}, []);
+
```

Comment:
Haven't seen these files for a while, so just wondering:

Why does it set the countdown date to `undefined` if the current date is less than or equal to the private sale end date? Does this mean there's no timer before and during the private sale?